### PR TITLE
EVG-2401: new simple.exec agent command

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: command/exec.go [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+	base
+}
+
+func simpleExecFactory() Command   { return &simpleExec{} }
+func (c *simpleExec) Name() string { return "simple.exec" }
+
+func (c *simpleExec) ParseParams(params map[string]interface{}) error {
+	err := mapstructure.Decode(params, c)
+	if err != nil {
+		return errors.Wrapf(err, "error decoding %v params", c.Name())
```
Comment (bsamek) [inline command/exec.go]: `%s`

File: command/exec.go [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+	}
+
+	if c.Command != "" {
+		if c.CommandName != "" || len(c.Args) > 0 {
```
Comment (bsamek) [inline command/exec.go]: Should `.c.CommandName` actually be `c.Command`? (based on the fact that that's what you're splitting on 77). I'm actually not sure what the difference between `command` and `command_name` is.

Comment (tychoish) [reply command/exec.go]: "command" is supposed to be a string form that we're will decompose into a binary name and a slice of args.

**Comment by tychoish** [comment]
Updated!

**Pull request merged**